plugins/ci/pkg/ci/ci.go

```diff
@@ -309,12 +311,23 @@ func (ci *CIScan) sendResults(reports []*models.ReportInfo) (*models.ScanResults
 			logrus.Fatalf("Unable to write contents for %s: %v", file.field, err)
 		}
 	}
-	w.Close()
 
 	repoDetails, err := getGitInfo(commands.ExecInDir, ci.config.Options.CIRunner, ci.repoBaseFolder, ci.config.Options.RepositoryName, ci.config.Options.BaseBranch)
 	if err != nil {
 		logrus.Fatalf("Unable to get git details: %v", err)
 	}
+	if len(repoDetails.filesModified) > 0 {
+		fw, err := w.CreateFormFile(filesModifiedFileName, filesModifiedFileName)
+		if err != nil {
+			logrus.Fatalf("Unable to create form for %s: %v", "files_modified", err)
+		}
+		_, err = fw.Write([]byte(strings.Join(repoDetails.filesModified, "\n")))
+		if err != nil {
+			logrus.Fatalf("Unable to write file for %s: %v", "files_modified", err)
+		}
+	}
+
+	w.Close()
```

Comment:
[opt] move `defer w.Close()` to line 298

Reply:
Still writing to it at this point

Reply:
That is what the `defer` would help.
The way it is set, if the program closes by any of `logrus.Fatalf` the writer would never be closed.